server/src/main/java/org/elasticsearch/common/geo/GeometryTreeReader.java

```diff
@@ -34,42 +34,50 @@
  */
 public class GeometryTreeReader implements ShapeTreeReader {
 
-    private final int extentOffset = 8;
+    private static final int EXTENT_OFFSET = 8;
+    private final int extentPosition;
```

Comment:
in the EdgeTree and such, this is called the `startPosition`. should we stay consistent here?

Reply:
Good catch! When I created this variable it was pointing to the start of extent.